landbosse/model/CollectionCost.py

```diff
@@ -364,12 +363,14 @@ def calc_num_turb_partial_strings(num_leftover_turb, num_turb_per_cable):
 
         Returns
         -------
-        self.output_dict['perc_partial_string'] : list
-            List of percent of turbines per cable type on partial string
+        np.array
```

Comment:
Changed from `self.output_dict['perf_partial_string']` because this method does not access `self` for its return value.